### DISCUSSION TITLE
Mastodon account for this project

### DISCUSSION THREAD

**Discussion by linux-lukas** [obsproject/obs-studio#5407] [General]
Would a Mastodon account for this project be possible?

**Comment by tt2468** [comment]
... but why?

**Reply by linux-lukas** [reply]
In terms of content, project announcements & project information and occasionally interaction with the Fediverse community would be topics, would be my idea. 

Why in the "Discussion" section? Because in my opinion it was the best thematic fit.

**Comment by WizardCM** [comment]
As it stands, our forums and Discord are the official channels where users can ask for help. For information about updates, users can 'Watch' the repository on GitHub, or follow us on Twitter.

We also allow people to talk on our subreddit, where we also post release notes.

We don't have a team big enough to manage yet another social presence, and can currently see no benefits of Mastodon for our use case.

**Reply by linux-lukas** [reply]
Unfortunately, I'm neither a friend of Discord nor of Reddit, because privacy issues. 

You could crosspost Twitter and Mastodon via moa.party?

Mastodon would mean not being dependent on a single company, like Facebook, Instagram and Twitter.

**Comment by Fenrirthviti** [comment]
We do not have an interest in maintaining or managing a presence on Mastodon at this time. We frequently get asked to support whatever new open or privacy-centric platform that comes around, but many of them are difficult to find, have small user bases, and are not really suited for the general end-user. We still maintain a barebones IRC presence on Quakenet and Libera if you do not want to use Discord, and there is always our Forums which can be used.
